### PR TITLE
Add Firestore to README.md.

### PR DESCRIPTION
**Pull request by ehsannas**

### CODE REVIEW

File: README.md
Comment (var-const) [inline README.md:71]: Hmm, I'm not sure if we need `Protobuf`, can you clarify?

Comment (ehsannas) [reply README.md:71]: I had to run `brew install protobuf` (and `sudo apt-get install protobuf-compiler` on Linux) when I first wanted to compile the firebase-cpp-sdk repo (although, now that I think about it, I built all the targets... perhaps if I had only built the `firebase_firestore` target alone it might have not been needed?)

Comment (var-const) [reply README.md:71]: We do use `Protobuf` in the iOS repo (in integration tests), maybe that's the reason?

It would be best to temporarily uninstall `protobuf` (or rename the executable, etc.) and try again, can you please try that?

Comment (ehsannas) [reply README.md:71]: I tried this, and the compilation was successful without it. So I'll update the readme

**Review by var-const — APPROVED** [review]
LGTM -- can you please also get an approval from somebody from the Games team before merging?

**Comment by ehsannas** [comment]
@a-maurice Would you please review this PR as well? Thanks

**Review by a-maurice — APPROVED** [review]

**Pull request merged**
